test/shared_setup.py

```diff
@@ -15,10 +16,52 @@ def test_get_name_url(self):
             'download_url': 'https://github.com/hpcugent/vsc-install/archive/0.1.2.tar.gz',
         }
         for fn in ['PKG-INFO', 'git_config', 'git_config_1', 'git_config_2', 'git_config_3', 'git_config_4']:
-            self.assertEqual(get_name_url(os.path.join(REPO_TEST_DIR, 'setup', fn), version='0.1.2'), res,
+            self.assertEqual(get_name_url(os.path.join(shared_setup.REPO_TEST_DIR, 'setup', fn), version='0.1.2'), res,
                              msg='determined name and url from %s file' % fn)
 
         res.pop('download_url')
         fn = 'git_config'
-        self.assertEqual(get_name_url(os.path.join(REPO_TEST_DIR, 'setup', fn), version='0.1.2', license_name='LGPLv2+'), res,
+        self.assertEqual(get_name_url(os.path.join(shared_setup.REPO_TEST_DIR, 'setup', fn), version='0.1.2', license_name='LGPLv2+'), res,
                          msg='determined name and url from %s file with license' % fn)
+
+    def test_sanitize(self):
+        """Test sanitize function"""
```

Comment:
Some python packages are named with a `python-` prefix already. e.g. [`python-jenkins`](https://pypi.python.org/pypi/python-jenkins/0.4.11), [`python-etcd`](https://pypi.python.org/pypi/python-etcd/0.4.2). Please put in a test with these to make sure we do or don't sanitize to `python-python-jenkins` (I don't know which is better/worse, but it would be good to have it documented and tested so we can catch unexpected changes).